content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Linux पर bash ऑटो-कम्पलीशन"
+description: "Linux पर bash ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक विन्विन्यास।"
+headless: true
+---
+
+## परिचय
+
+Bash के लिए kubectl  कम्पलीशन स्क्रिप्ट `kubectl completion bash` कमांड के साथ उत्पन्न की जा सकती है। आपके शेल में कम्पलीशन स्क्रिप्ट को सोर्स करने से kubectl ऑटोकम्पलीशन ऐनेबल हो जाती है।
```

Comment:
I believe कम्पलीशन can be replaced by समापन? Or are we trying not to stray too far away from the terminology that is being introduced?

Reply:
if I replace  कम्पलीशन by समापन, then what about this auto-completion, bash-completion ?